lib/atoms/div/metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "ln-a-img",
+  "description": "Atom - Img",
+  "params": {}
+}

```

Comment:
I think we don't need this atom, because "div" will generally be a container of other components.